test-apps/appui-test-app/appui-test-providers/src/ui/buttons/ViewSelectorPanel.tsx

```diff
@@ -5,11 +5,11 @@
 
 import * as React from "react";
 import "./ViewSelectorPanel.scss";
-import { getListPanel, ListItem, ListItemType, SupportsViewSelectorChange, UiFramework, useActiveViewport, ViewUtilities } from "@itwin/appui-react";
+import { getListPanel, IModelConnectedViewSelector, ListItem, ListItemType, SupportsViewSelectorChange, UiFramework, useActiveViewport, ViewUtilities } from "@itwin/appui-react";
 import { IModelApp, IModelConnection, Viewport } from "@itwin/core-frontend";
 import { CustomToolbarItem } from "@itwin/components-react";
 
-// eslint-disable-next-line @typescript-eslint/naming-convention
+// eslint-disable-next-line @typescript-eslint/naming-convention, @typescript-eslint/no-unused-vars
```

Comment:
which vars are unused? maybe they should be removed if theyre not being used?

Reply:
The whole ViewSelectorPanel in this test app isn't being used anymore. ViewSelectorPanel only exists in the test app whereas ViewSelector is the component in the appui library. I wasn't sure if we wanted to get rid of the entire ViewSelectorPanel, so I just left it for now.

Reply:
ill leave to the ui team to comment what to do here

Reply:
Please remove the unused stuff, if we now have a component that does the job, we'll prefer people peaking here to use the right component instead of creating a new one.

Reply:
Looks like you changed it to render the component with the new panel, but left the definition for the old panel in place. Since this is test code, you can simply remove it.